framework/global/settings.cpp

```diff
@@ -69,6 +69,10 @@ void Settings::reload()
 void Settings::load()
 {
     m_items = readItems();
+
+    if (source()) {
+        source()->sourceChanged().onNotify(this, [this]() { reload(); });
```

Comment:
Do we only have a global change notifier? Do we want to make a notifier for changing one value? like in Settings class

Reply:
This notifier doesn't notify about changes in values. Its goal to notify when the source was changed (for example, a workspace was switched to another one)